pkg/build/builder/daemonless.go

```diff
@@ -212,6 +212,25 @@ func daemonlessProcessLimits() (defaultProcessLimits []string) {
 	return defaultProcessLimits
 }
 
+func copyRHSMData(source, destination string) error {
```

Comment:
We have similar "copy directory contents" in https://github.com/openshift/builder/blob/master/cmd/main.go#L74-L126. Can we consolidate into a utility package?

Reply:
> We have similar "copy directory contents" in https://github.com/openshift/builder/blob/master/cmd/main.go#L74-L126. Can we consolidate into a utility package?

update for this ^^ pushed @adambkaplan

Reply:
Do we test this with directories that contain symlinks to directories?  I think `ioutil.ReadDir()` uses `lstat()`, which would return `os.FileInfo`s that returns `false` from `IsDir()`.

Reply:
Yeah this aspect came up when researching golang recursive copy, but I stopped short of adding that complexity since `/run/secret/rhsm` does not have symlinks based on my testing, so I stopped short of changing this existing code path.

But I'll defer to you @nalind and @adambkaplan on if we want to add this in just in case.